src/main/flight/mixer.c

```diff
@@ -245,7 +244,7 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 #if defined(USE_BATTERY_VOLTAGE_SAG_COMPENSATION)
         float motorRangeAttenuationFactor = 0;
         // reduce motorRangeMax when battery is full
-        if (mixerRuntime.vbatSagCompensationFactor > 0.0f) {
+        if (!mixerConfig()->rpm_limiter && mixerRuntime.vbatSagCompensationFactor > 0.0f) {
```

Comment:
i think this will fail to compile if USE_RPM_LIMITER is undefined
Suggestion: can make something like that:
```
#ifdef USE_RPM_LIMITER
    #define RPM_LIMIT_ACTIVE mixerConfig()->rpm_limiter
#else
    #define RPM_LIMIT_ACTIVE false
#endif
```
and then use `RPM_LIMIT_ACTIVE` anywhere in the code safely